.releaserc

```diff
@@ -1,17 +1,19 @@
 {
   "branch": "master",
   "tagFormat": "java-sdk-${version}",
+  "debug": true,
```

Comment:
technically this doesn't need to be here... I think the debug flag is really just used to produce more verbose output in the travis build log (possibly... presumably...  :) ).

Reply:
Yep, I went ahead and added it _just_ in case this change still isn't it so that I can fix it easier 😬